components/schema-migrator/run.sh

```diff
@@ -36,11 +36,32 @@ if [ "${dbReady}" != true ] ; then
     exit 1
 fi
 
+DB_NAME_SSL=$DB_NAME
+SSL_OPTION=""
 if [ -n "${DB_SSL}" ] ; then
-  DB_NAME="${DB_NAME}?sslmode=${DB_SSL}"
+  DB_NAME_SSL="${DB_NAME}?sslmode=${DB_SSL}"
+  SSL_OPTION="sslmode=${DB_SSL}"
 fi
 
-CONNECTION_STRING="postgres://$DB_USER:$DB_PASSWORD@$DB_HOST:$DB_PORT/$DB_NAME"
+
+if [[ -f seeds/dump.sql ]]; then
+    echo "Will reuse existing dump in seeds/dump.sql"
+    cat seeds/dump.sql | \
+        PGPASSWORD="${DB_PASSWORD}" psql -U "${DB_USER}" -h "${DB_HOST}" -p "${DB_PORT}" -d "${DB_NAME}" --set="${SSL_OPTION}"
+
+    REMOTE_MIGRATION_VERSION=$(PGPASSWORD="${DB_PASSWORD}" psql -qtAX -U "${DB_USER}" -h "${DB_HOST}" -p "${DB_PORT}" -d "${DB_NAME}" -c "SELECT version FROM schema_migrations")
+    LOCAL_MIGRATION_VERSION=$(echo $(ls migrations/director | tail -n 1) | grep -o -E '[0-9]+' | head -1 | sed -e 's/^0\+//')
+
+    if [[ ${REMOTE_MIGRATION_VERSION} = ${LOCAL_MIGRATION_VERSION} ]]; then
```

Comment:
Maybe add coloring to messages displayed in this if's body just like you've done in director/run.sh. Looks quite good to me :D